src/Config/Schema.php

```diff
@@ -61,291 +89,44 @@ public function getDefaults(string $stage): array
                     StageConfigInterface::STAGE_GLOBAL,
                     StageConfigInterface::STAGE_BUILD,
                 ],
                 self::SCHEMA_DEFAULT_VALUE => [StageConfigInterface::STAGE_BUILD => 1]
             ],
-            DeployInterface::VAR_LOCK_PROVIDER => [
-                self::SCHEMA_TYPE => ['string'],
-                self::SCHEMA_STAGE => [
-                    StageConfigInterface::STAGE_GLOBAL,
-                    StageConfigInterface::STAGE_DEPLOY
-                ],
-                self::SCHEMA_VALUE_VALIDATION => ['db', 'file'],
-                self::SCHEMA_DEFAULT_VALUE => [
-                    StageConfigInterface::STAGE_DEPLOY => 'file',
-                ],
-            ],
-            DeployInterface::VAR_REDIS_USE_SLAVE_CONNECTION => [
-                self::SCHEMA_TYPE => ['boolean'],
-                self::SCHEMA_STAGE => [
-                    StageConfigInterface::STAGE_GLOBAL,
-                    StageConfigInterface::STAGE_DEPLOY
-                ],
-                self::SCHEMA_DEFAULT_VALUE => [
-                    StageConfigInterface::STAGE_DEPLOY => false,
-                ],
-            ],
-            DeployInterface::VAR_MYSQL_USE_SLAVE_CONNECTION => [
-                self::SCHEMA_TYPE => ['boolean'],
-                self::SCHEMA_STAGE => [
-                    StageConfigInterface::STAGE_GLOBAL,
-                    StageConfigInterface::STAGE_DEPLOY
-                ],
-                self::SCHEMA_DEFAULT_VALUE => [
-                    StageConfigInterface::STAGE_DEPLOY => false,
-                ],
-            ],
-            DeployInterface::VAR_UPDATE_URLS => [
-                self::SCHEMA_TYPE => ['boolean'],
-                self::SCHEMA_STAGE => [
-                    StageConfigInterface::STAGE_GLOBAL,
-                    StageConfigInterface::STAGE_DEPLOY
-                ],
-                self::SCHEMA_DEFAULT_VALUE => [
-                    StageConfigInterface::STAGE_DEPLOY => true,
-                ],
-            ],
-            DeployInterface::VAR_FORCE_UPDATE_URLS => [
-                self::SCHEMA_TYPE => ['boolean'],
-                self::SCHEMA_STAGE => [
-                    StageConfigInterface::STAGE_GLOBAL,
-                    StageConfigInterface::STAGE_DEPLOY
-                ],
-                self::SCHEMA_DEFAULT_VALUE => [
-                    StageConfigInterface::STAGE_DEPLOY => false,
-                ],
-            ],
-            DeployInterface::VAR_CLEAN_STATIC_FILES => [
-                self::SCHEMA_TYPE => ['boolean'],
-                self::SCHEMA_STAGE => [
-                    StageConfigInterface::STAGE_GLOBAL,
-                    StageConfigInterface::STAGE_DEPLOY
-                ],
-                self::SCHEMA_DEFAULT_VALUE => [
-                    StageConfigInterface::STAGE_DEPLOY => true,
-                ],
-            ],
             DeployInterface::VAR_SEARCH_CONFIGURATION => [
```

Comment:
Why do we keep some option configuration here? It should be moved to `.yaml` file

Reply:
Will be moved in next task